rites/roman1969/__tests__/christmastide-psalter.test.ts

```diff
@@ -0,0 +1,43 @@
+import { LiturgicalCalendar, Romcal } from '@src/rite-roman1969';
+
+import { cache } from './fixtures/lh.kbs.sk.fixture';
+
+describe('Psalter weeks after Christmas', () => {
+  let romcal: LiturgicalCalendar;
+
+  beforeAll(async () => {
+    romcal = await new Romcal({ scope: 'liturgical' }).generateCalendar(2025);
```

Comment:
We might want to add s test for the `gregorian` scope.

Reply:
feel free